k8s/deploy

```diff
@@ -225,3 +225,21 @@ isAllowedDeployEnvironment() {
     [ -z $(echo ${ALLOWED_DEPLOY_ENVIRONMENTS[@]} | grep -o $__environment) ] && error "$__environment is not an allowed deployment environment"
     success "Setting up deployments for $__environment environment"
 }
+
+configureSlackNotifications() {
+    
+    EMOJIS=(":celebrate:"  ":party_dinosaur:" ":hammer-time:"  ":andela:" ":victory-danch:" ":aw-yeah:" ":carlton-dance:" ":partyparrot:" ":dancing-penguin:" ":aww-yeah-remix:" )
+
+    RANDOM=$$$(date +%s)
+    EMOJI=${EMOJIS[$RANDOM % ${#EMOJIS[@]} ]}
+    COMMIT_LINK="https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/${CIRCLE_BRANCH}/commit/${CIRCLE_SHA1}"
```

Comment:
looks like **COMMIT_LINK** is not in use, is it necessary?

Reply:
Done. I have referred to the commit_link as the build details within the deployment text